### PR TITLE
Gyanendra dev

### PR DESCRIPTION
**Pull request by devGyan**

### CODE REVIEW

File: client/src/ActionType.js [hunk 1]
```diff
@@ -3,4 +3,5 @@ export const DARK_THEME_INACTIVE = 'DARK_THEME_INACTIVE';
 export const SIGNED_IN_SUCCEEDED = 'SIGNED_IN_SUCCEEDED';
 export const RESET_SIGNED_IN_SUCCEEDED = 'RESET_SIGNED_IN_SUCCEEDED';
 export const USER_INFO = 'USER_INFO';
+export const MILITARY_VERSION = 'MILITARY_VERSION';
```
Comment (rhk12) [inline client/src/ActionType.js]: You can add SPORTS_VERSION variable too.

File: client/src/components/CommanderTeamView.js [hunk 2]
```diff
@@ -4,6 +4,8 @@ import Footer from './Footer';
 import PenstateUniversity from './PenstateUniversity';
```
Comment (rhk12) [inline client/src/components/CommanderTeamView.js]: @eralpsahin @devGyan  What is this variable for?

File: client/src/components/Footer.js [hunk 3]
```diff
@@ -57,9 +60,10 @@ class Footer extends React.Component {
           </div>
           <div className="row text-center">
             <div className="col-md-12 col-lg-12">
-              <p onClick={this.redirectToOther} className="copyright">
-                © 2019 Copyright: Penn State Computational Biomechanics Group
-              </p>
+            <p>
+                    Contact Us: info@NSFCAREER.IO <br />
+                    <span onClick={()=>window.open('')}>  IP </span> | Privacy Policy &amp; <span onClick={()=>window.open('')}>  IRB </span> | Collaborate
```
Comment (rhk12) [inline client/src/components/Footer.js]: I said you need to create new pages, not just open new windows. Make pages for these.

File: client/src/components/CommanderTeamView.js (hunk 2 above)
Comment (devGyan) [reply client/src/components/CommanderTeamView.js]: It is the reusable component.
Screenshot from 2019-09-19 10-06-53

Comment (devGyan) [reply client/src/components/CommanderTeamView.js]: it is used in team admin as well as organization admin.

File: client/src/components/Footer.js (hunk 3 above)
Comment (devGyan) [reply client/src/components/Footer.js]: I apologize for this. but I don't have the design of the page. could you please provide.

File: client/src/ActionType.js (hunk 1 above)
Comment (devGyan) [reply client/src/ActionType.js]: sure I will add it.

Comment (devGyan) [reply client/src/ActionType.js]: added

**Comment by eralpsahin** [comment]
> @devGyan @eralpsahin this pull currently has conflicts with master. can you fix?

It should be fixed now.

**Pull request merged**